.prettierignore

```diff
@@ -1 +1,2 @@
-pnpm-lock.yaml
\ No newline at end of file
+pnpm-lock.yaml
+packages/lazy-fn-vite-plugin/tests/sources/*
```

Comment:
```suggestion
```

This has been already excluded in `packages/lazy-fn-vite-plugin/.prettierignore` so I think this can be removed from there.

Reply:
I don't know the reason, but my IDE still complain without it